plugins/node.d/df_inode

```diff
@@ -30,7 +32,7 @@ The following environment variables are used by this plugin:
 . "$MUNIN_LIBDIR/plugins/plugin.sh"
 
 
-df_entries=$(df -P -l -i -x btrfs 2>/dev/null | sed -e '1d' -e '/\/\//d' -e 's/%//' | sort)
+df_entries=$(df -P -l -i -x btrfs 2>/dev/null | sed -e '1d' -e '/\/\//d' -e 's/%//' | sort) || true
```

Comment:
See above.

Reply:
we do not want to terminate the script here (with the introduction of `set -e`)

Reply:
Yes, I understand. But I cannot imagine, which part of this line could fail (for a good reason that we want to ignore). What do you think?